_infra/docker/Dockerfile

```diff
@@ -1,13 +1,16 @@
 # Dockerfile specifically for kubernetes
 FROM python:3.11-slim
 
-RUN apt update && apt install -y build-essential curl gunicorn
+RUN apt update && apt install -y build-essential curl gunicorn procps
```

Comment:
This is the only way I've found to install `ps` now that we are the new user - it will be removed after testing